src/node_perf.cc

```diff
@@ -263,6 +263,14 @@ void LoopIdleTime(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(1.0 * idle_time / NANOS_PER_MILLIS);
 }
 
+static double FastLoopIdleTime(v8::Local<v8::Value> receiver) {
+  Environment* env = Environment::GetCurrent(v8::Isolate::GetCurrent());
```

Comment:
You need to add FastAPiCallbackOptions& as a second parameter

```suggestion
  Environment* env = Environment::GetCurrent(options.isolate);
```